### PR TITLE
fix: [M3-8937] - Content shifting on Linode Details summary graphs

### PR DESCRIPTION
**Pull request by bnussman-akamai**
## Description 📝

Quick PR to try to improve the content shifting when loading a Linode Details page. It is pretty bad right now 😖

- Improved loading state so that there is less content shifting 🔄📈 
- Cleaned up styled components / one-off styles 🧹 
- Improved dark mode styles 🌑 

## Preview 📷

| | Before  | After   |
| ------- | ------- | ------- |
| Improved loading state | <video src="https://github.com/user-attachments/assets/16486e6e-fedc-428f-82c6-b32b9c081817" /> | <video src="https://github.com/user-attachments/assets/b110d4cb-0f7c-40ef-a90c-d338b9725e77" /> |
| Improved dark mode | Screenshot 2024-11-21 at 10 07 47 AM    |  Screenshot 2024-11-21 at 10 07 43 AM | 

## How to test 🧪

- Test the Linode Details page summary (graphs) section for style  and functionality regressions 👀 📈 git

## As an Author, I have considered 🤔

- 👀 Doing a self review
- ❔ Our [contribution guidelines](https://github.com/linode/manager/blob/develop/docs/CONTRIBUTING.md)
- 🤏 Splitting feature into small PRs
- ➕ Adding a [changeset](https://github.com/linode/manager/blob/develop/docs/CONTRIBUTING.md#writing-a-changeset)
- 🧪 Providing/improving test coverage
- 🔐 Removing all sensitive information from the code and PR description
- 🚩 Using a feature flag to protect the release
- 👣 Providing comprehensive reproduction steps
- 📑 Providing or updating our documentation
- 🕛 Scheduling a pair reviewing session
- 📱 Providing mobile support
- ♿  Providing accessibility support

<br/>

- [ ] I have read and considered all applicable items listed above.

## As an Author, before moving this PR from Draft to Open, I confirmed ✅

- [ ] All unit tests are passing
- [ ] TypeScript compilation succeeded without errors
- [ ] Code passes all linting rules

### CODE REVIEW

File: packages/manager/src/features/Linodes/LinodesDetail/LinodeSummary/LinodeSummary.tsx [hunk 1]
```diff
@@ -40,15 +40,11 @@ import type { Item } from 'src/components/EnhancedSelect/Select';
 setUpCharts();
 
 interface Props {
-  isBareMetalInstance: boolean;
```
Comment (dwiley-akamai) [inline packages/manager/src/features/Linodes/LinodesDetail/LinodeSummary/LinodeSummary.tsx:43]: Not sure about removing the bare metal checks in this file 🤔

**Review by dwiley-akamai — COMMENTED** [review]
For some reason, the graphs aren't rendering for me on this branch

https://github.com/user-attachments/assets/caa94635-8364-4740-83d0-9eebf387526d

**Comment by bnussman-akamai** [comment]
@dwiley-akamai I think [this change](https://github.com/linode/manager/pull/11204/files#diff-2ec2b252bddc9d4a5748565cdd82e62baa9f6c594258111117e48d6fb3637da4R257) caused that when I merged in the latest `develop`. I made a change reverting that change (and had to update a few cypress tests)

**Review by dwiley-akamai — APPROVED** [review]
Graph boxes don't jump in size once the graph fully loads ✅

One thing I did notice is at the smallest viewport width, upon first loading the page, the placement of the graphs is wonky (in prod, they stack as expected):

<details>
<summary>Screenshot</summary>

Screenshot 2024-11-22 at 5 35 44 PM
</details>

**Review by jaalah-akamai — APPROVED** [review]
✅ Responsive Behavior
✅ Works with other charts (Cloud Pulse)
✅ Dark Mode 🔥 
👀 Was not noticing what @dwiley-akamai was observing

**Review by coliu-akamai — COMMENTED** [review]
Noticing the same as Dajahi + sometimes the graphs don't adjust to the screen's width when going from smaller to larger widths. I'm on chrome - will try testing on other browsers

https://github.com/user-attachments/assets/1e02e16a-70b6-4d63-961f-0f9675d80b29

**Comment by bnussman-akamai** [comment]
@coliu-akamai @dwiley-akamai Good catch. I pushed 29c492b to hopefully fix that issue. Let me know if that worked

**Review by coliu-akamai — APPROVED** [review]
worked for me :thxbanks: 🚀

**Pull request merged**